### PR TITLE
first attempt at automated travis upload to launchpad

### PR DESCRIPTION
**Pull request by maqifrnswa**
see https://forum.openmw.org/viewtopic.php?f=20&t=3792

This will have to be merged with master to even begin testing if it works since all secrets are signed with OpenMW/openmw's public key and pull requests skip the deploy step. Good news is that if deploy fails, the build testing won't.

If it works, then our daily build PPA will automatically be tied to travis, and not be dependent on other systems

EDIT: this uses bzr, but launchpad supports git now. I won't have time for a while, but it's probably easier to use this script to just mirror github to launchpad

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -15,6 +15,9 @@ env:
    # The next declaration is the encrypted COVERITY_SCAN_TOKEN, created
    #   via the "travis encrypt" command using the project repo's public key
    - secure: "jybGzAdUbqt9vWR/GEnRd96BgAi/7Zd1+2HK68j/i/8+/1YH2XxLOy4Jv/DUBhBlJIkxs/Xv8dRcUlFOclZDHX1d/9Qnsqd3oUVkD7k1y7cTOWy9TBQaE/v/kZo3LpzA3xPwwthrb0BvqIbOfIELi5fS5s8ba85WFRg3AX70wWE="
+   # Next the DEPLOY_PASS and BZR_LOGIN tokens
```
Comment (kpp) [inline .travis.yml]: BZR?

Comment (psi29a) [inline .travis.yml]: yes... launchpad uses bazaar but it is slowly being deprecated for git.

**Comment by maqifrnswa** [comment]
can close this request! launchpad allows git imports directly, it's up and running at:
https://code.launchpad.net/~openmw/+recipe/openmw-daily-git
@psi29a can commit to the debian repo to update the packaging, and is an owner of the project and branches on launchpad
